File: docs/cs/src/model/Factura.cs
using System;
using System.Collections.Generic;

namespace Model
{
    public class Factura
    {
        public long IdFactura { get; set; }
        public double Monto { get; set; }
        public DateTime Fecha { get; set; }
        public bool Pendiente { get; set; }
        public string TipoPago { get; set; }
        public Cliente Cliente { get; set; }
        public Empleado Empleado { get; set; }
        public List<Producto> Productos { get; set; }
        public List<Servicio> Servicios { get; set; }
    }
}

File: Proyecto kotlin PeluPos/proyectobase2425/app/src/main/java/com/proyecto/PeluPos/models/Factura.kt
import java.util.Date

data class Factura(
    val idFactura: Long,
    var monto: Double = 0.0,
    val fecha: Date = Date(),
    var pendiente: Boolean = true,
    var tipoPago: String = "",
    val cliente: Cliente,
    val empleado: Empleado,
    val productos: MutableList<Producto> = mutableListOf(),
    val servicios: MutableList<Servicio> = mutableListOf()
)